app/utils.py

```diff
@@ -192,26 +219,39 @@ def validate_uk_mobile_phone_number(number, locale):
 
         number = ProcessMobileNumber.normalise_phone_number(number, locale).lstrip(uk_prefix).lstrip('0')
 
+        if len(number) == 0:
+            if locale == 'cy':
+                # TODO: Add Welsh Translation
+                raise InvalidDataErrorWelsh('Enter your mobile number', message_type='empty')
+            else:
+                raise InvalidDataError('Enter your mobile number', message_type='empty')
+
         if not number.startswith('7'):
             if locale == 'cy':
                 # TODO: Add Welsh Translation
-                raise InvalidDataErrorWelsh('The mobile phone number is not a UK mobile number')
+                raise InvalidDataErrorWelsh('Enter a UK mobile number in a valid format, for example, '
```

Comment:
Not very DRY. For repetitively used string literals or constant we could declare Them as a variable at the start of the module or even import them from __init__.py?